Makefile

```diff
@@ -14,9 +14,12 @@ install_proj: install_proj_deps ## Install PROJ on system (avoid PROJ recompilin
 
 	wget https://github.com/OSGeo/proj.4/releases/download/$(PROJ_VERSION)/proj-$(PROJ_VERSION).tar.gz
 	tar -xzvf proj-$(PROJ_VERSION).tar.gz
-	cd proj-$(PROJ_VERSION) && ./configure --prefix=/usr
-	make -C proj-$(PROJ_VERSION)
-	sudo make install -C proj-$(PROJ_VERSION)
+	cd proj-$(PROJ_VERSION) \
+	&& mkdir build \
```

Comment:
No need for `./configure` anymore? That's surprising.

Reply:
they switched from `make` to `cmake`